codice/ddf
# DDF-4070 Rename log4j2.config.xml to the more standard log4j2.xml

#### What does this PR do?
Renames the `etc/log4j2.config.xml` configuration file to `etc/log4j2.xml` in order to follow the standards set by the Karaf and Log4j2 documentation.

#### Who is reviewing it? 
@Kjames5269 
@emmberk 

#### Ask 2 committers to review/merge the PR and tag them here.
@clockard
@coyotesqrl

#### How should this be tested?
1. Build and unzip DDF.
2. Follow the [Enabling Fallback Audit Logging documentation (screenshot below) to configure DDF to use the `etc/log4j2.xml` file.
2. Verify all of the CLI logging commands work properly (`log:display`, `log:tail`). **Note that** in Karaf 4.1.2, `log:set` and `log:get` are broken for the XML configuration of `log4j2`, and as such those commands should **not** work for this hero.

![image](https://user-images.githubusercontent.com/14284361/44422810-075c2b80-a539-11e8-9306-f31f18bd10ec.png)

#### What are the relevant tickets?
[DDF-4070](https://codice.atlassian.net/browse/DDF-4070)

#### Checklist:

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.
